pkgs/by-name/al/alice-tools/package.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  stdenv,
+  nix-update-script,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  bison,
+  flex,
+  libiconv,
+  libpng,
+  libjpeg,
+  libwebp,
+  zlib,
+  versionCheckHook,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "alice-tools";
+  version = "0.13.0";
+
+  src = fetchFromGitHub {
+    owner = "nunuhara";
+    repo = "alice-tools";
+    tag = finalAttrs.version;
+    fetchSubmodules = true;
+    hash = "sha256-DazWnBeI5XShkIx41GFZLP3BbE0O8T9uflvKIZUXCHo=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    bison
+    flex
+  ];
+
+  buildInputs = [
+    libiconv
+    libpng
+    libjpeg
+    libwebp
+    zlib
+  ];
+
+  # Default install step only installs a static library of a build dependency
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 src/alice $out/bin/alice
+
+    runHook postInstall
+  '';
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = "--version";
+  doInstallCheck = true;
+
+  passthru.updateScript = nix-update-script { };
```

There was no reason to change this. Don't force this on me.

```suggestion
  passthru.updateScript = gitUpdater { };
```

---

I haven't had any issue with `gitUpdater` so far, but I *have* had issues with `nix-update-script` in the past. I also prefer using our in-tree solution over using external projects that require syncing.

By all means, please use `nix-update` and make such a change to derivations that you maintain. But why change smth that isn't broken.